.github/workflows/validate_ynamazon.yml

```diff
@@ -0,0 +1,28 @@
+name: YNAmazon
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install libjpeg-dev (used for Pillow library)
+        run: sudo apt-get install libjpeg-dev
```

Comment:
This one I didn't like (it's slow to run), but it does work without needing to manage a separate Dockerfile / container lifecycle.